cla-v1.1.md

```diff
@@ -9,7 +9,7 @@ CONTRIBUTOR, AS WELL AS FOR THE PROTECTION OF QLIK AND ITS USERS AND
 DOES NOT AFFECT YOUR RIGHTS TO USE YOUR OWN CONTRIBUTIONS FOR ANY OTHER 
 PURPOSE. YOU RESERVE ALL RIGHT, TITLE AND INTEREST IN AND TO YOUR 
 CONTRIBUTIONS. PLEASE COMPLETE AND SIGN THIS AGREEMENT THEN SCAN AND 
```

Comment:
By having `SCAN AND SEND TO QLIK`, doesn't that imply people have to print it, write a signature, scan it in, and e-mail it? The process today is: Do a PR with the info described in `sign-cla.md`, then e-mail the additional info described later on the same page.